config/routes.rb

```diff
@@ -140,6 +140,10 @@
       end
     end
 
+    if IdentityConfig.store.select_multiple_mfa_options
+      get '/auth_method_confirmation' => 'mfa_confirmation#index'
```

Comment:
A few thoughts on this route and associated controller:

- `MfaConfirmationController` seems to be primarily concerned with the "reauthn" (password reentry) behavior, not with MFA setup, where MFA setup is split across `TwoFactorAuthenticationSetupController` and ad-hoc controllers. I might wonder if this should be a separate controller, or part of `TwoFactorAuthenticationSetupController`?
- While we're not entirely consistent about it, I usually prefer when the route aligns to the controller name, since it makes it easier for me to backtrack from a URL to its associated controller.
- To me, `#index` implies some sort of listing, or at least an entry-point of a controller, where this is more like a confirmation screen.
 
Depending on how this feature evolves, I suppose this view would probably be the rendered result of, for example, `TwoFactorAuthenticationSetupController#create` ? With that in mind, it could be fine if it's expected to be temporary, though we may be able to save some future effort by implementing it closer to where we expect it to be eventually used.

Reply:
- I think we are going in the direction of using the `TwoFactorAuthenticationSetupController`  to set up the MFA prototype. [In fact, work for setup in that controller is being done in a separate ticket](https://github.com/18F/identity-idp/pull/6138/files#diff-f36e070420970b2439a54dad918e009cc6f4ca0ad273c7b0624c3d81a7bfbfb8)
- I think the plan is to have the route and controller to reflect its new functionality. This might be in the form of changing the name of `TwoFactorAuthenticationSetupController` and any routes associated with it.
- By the look of it, it can be changed to a `#show`. I can address that part now.

I anticipate that we will have additional changes to routes and controllers. There are a few things to think through to make it sound, but I think that is a little further down the road.

Reply:
That all sounds good to me! 👍